cypress/objects/taskPage.js

```diff
@@ -0,0 +1,19 @@
+class TaskPage {
+    getCreatTaskButton() {
+        return cy.get('.btn[type="button"]')
+    }
+
+    getTaskNameField() {
```

Comment:
suggest using id`s instead of class names if possible - cy.get('[id="taskDetailsForm.name"]')